local/sites/test-site/inventory.yml

```diff
@@ -15,12 +19,19 @@ name: etcd2
 interfaces:
   - type: physical
     subnet: vagrant1
+    bmc:
+      type: ssh
+      username: core
+      keypath: /vagrant/provision_files/id_rsa
     mac: 00:00:00:00:00:22
     ipv4: 10.10.10.22
 metadata:
     mstr: etcd2
     role: kube-master
     ip: 10.10.10.22
+    ssh_authorized_keys:
```

Comment:
see above, I want the ignition `etcd-proxy` machine to be our one example on how to do a host level ssh_key declaration, the others should pull from the site level `env.yml`